### DISCUSSION TITLE
Removal of KitPvP eCloud Expansion

### DISCUSSION THREAD

**Discussion by cervinakuy** [PlaceholderAPI/PlaceholderAPI#609] [(2) Support] [answered]
Hello there,

I'm not really sure if this is the correct place, so if not, please point me in the right direction.

Otherwise, I would like my [KitPvP eCloud expansion](https://api.extendedclip.com/expansions/kitpvp/) removed from the eCloud website if possible. I uploaded it in 2019 back when that was how PlaceholderAPI expansions worked (I believe, I can't really remember), and because of the size of KitPvP, many users are accidentally thinking this expansion is required for placeholders when it is not.

It would be very appreciated if it could be removed. I initially uploaded it with my own account onto the site, and I can definitely show proof of account ownership if necessary. Although it no longer says I authored it, now it is apparently owned by `outdated`. But regardless, any help would be very appreciated.

Thanks!

**Comment by darbyjack** [comment, marked as answer]
I have gone ahead and removed this expansion from the eCloud. Thanks for letting me know.
